coverage/.resultset.json

```diff
@@ -0,0 +1,305 @@
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null
+      ],
+      "/home/avest/Ragnarson/DayIV/lib/shop_basket.rb": [
+        1,
+        null,
+        1,
+        null,
+        1,
+        11,
+        10,
+        10,
+        null,
+        null,
+        1,
+        13,
+        11,
+        11,
+        null,
+        null,
+        1,
+        5,
+        10,
+        5,
+        4,
+        4,
+        null,
+        null,
+        null,
+        1,
+        2,
+        null,
+        null,
+        1,
+        9,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        null,
+        2,
+        null,
+        1,
+        1,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null
+      ],
+      "/home/avest/Ragnarson/DayIV/lib/warehouse.rb": [
+        1,
+        null,
+        1,
+        null,
+        1,
+        17,
+        31,
+        16,
+        null,
+        null,
+        1,
+        13,
+        11,
+        null,
+        null,
+        null,
+        1,
+        4,
+        null,
+        null,
+        1,
+        1,
+        13,
+        25,
+        null,
+        null,
+        null
+      ],
+      "/home/avest/Ragnarson/DayIV/spec/unit/warehouse_spec.rb": [
+        1,
+        1,
+        1,
+        null,
+        1,
+        null,
+        1,
+        null,
+        1,
+        1,
+        null,
+        null,
+        1,
+        1,
+        null,
+        null,
+        1,
+        2,
+        null,
+        null,
+        null,
+        1,
+        1,
+        2,
+        null,
+        null,
+        1,
+        2,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null,
+        null,
+        1,
+        1,
+        1,
+        1,
+        1,
+        null,
+        null,
+        1,
+        1,
+        1,
+        null,
+        null,
+        null
+      ]
+    },
+    "timestamp": 1467972347
+  }
```

Comment:
You should not commit those files, drop them from repo and add them to gitignore